setup.py

```diff
@@ -161,6 +161,7 @@ def get_extensions():
         define_macros += [('MMCV_WITH_TRT', None)]
         cuda_args = os.getenv('MMCV_CUDA_ARGS')
         extra_compile_args['nvcc'] = [cuda_args] if cuda_args else []
+        extra_compile_args['nvcc'] += ['-Xcompiler=-fno-gnu-unique']
```

Comment:
Hi @twmht , maybe we can add a comment here to explain why we need to add the flag

Reply:
sounds great. I would update the PR later.

Reply:
done

Reply:
suggest merging the master branch to your branch which will fix the error when building documentation.

Reply:
you mean that I need to merge upstream master to this branch? I thought the unit test is only related to upstream master with this PR, am i wrong?

Reply:
Yet, merge the upstream master to this branch.